### ISSUE TITLE
📋 Others — 2026-04-05 (48 papers)

### ISSUE THREAD

**Issue by DaehyeopKim** [DaehyeopKim/DaehyeopKim.github.io#60]
### 🏆 Oral

**1. [Algorithm Development in Neural Networks: Insights from the Streaming Parity Task](https://openreview.net/forum?id=3go0lhfxd0)**
  *Loek van Rossem, Andrew M Saxe*
  📍 ICML 2025 oral
  > Even when massively overparameterized, deep neural networks show a remarkable ability to generalize. Research on this phenomenon has focused on generalization within distribution, via smooth interp...

**2. [Common Corpus: The Largest Collection of Ethical Data for LLM Pre-Training](https://openreview.net/forum?id=0wSlFpMsGb)**
  *Pierre-Carl Langlais, Pavel Chizhov, Catherine Arnett et al.*
  📍 ICLR 2026 Oral
  > Large Language Models (LLMs) are pre-trained on large data from different sources and domains. These datasets often contain trillions of tokens, including large portions of copyrighted or proprieta...

**3. [Q-RAG: Long Context Multi‑Step Retrieval via Value‑Based Embedder Training](https://openreview.net/forum?id=MS9nWFY7LG)**
  *Artyom Sorokin, Nazar Buzun, Aleksandr Anokhin et al.*
  📍 ICLR 2026 Oral
  > Retrieval-Augmented Generation (RAG) methods enhance LLM performance by efficiently filtering relevant context for LLMs, reducing hallucinations and inference cost.
However, most existing RAG metho...

**4. [DarkBench: Benchmarking Dark Patterns in Large Language Models](https://openreview.net/forum?id=odjMSBSWRt)**
  *Esben Kran, Hieu Minh Nguyen, Akash Kundu et al.*
  📍 ICLR 2025 Oral
  > We introduce DarkBench, a comprehensive benchmark for detecting dark design patterns—manipulative techniques that influence user behavior—in interactions with large language models (LLMs). Our benc...

**5. [Chemist-aligned retrosynthesis by ensembling diverse inductive bias models](https://openreview.net/forum?id=F05pziBqOg)**
  *Krzysztof Maziarz, Guoqing Liu, Austin Tripp et al.*
  📍 NeurIPS2025-AI4Science Oral
  > Chemical synthesis remains a critical bottleneck in the discovery and manufacture of functional small molecules. AI-based synthesis planning models could be a potential remedy to find effective syn...

**6. [Editing Large Language Models via Adaptive Gradient Guidance](https://openreview.net/forum?id=q8fJI2r1I8)**
  *Xiaojie Gu, Guangxu Chen, Shuliang Liu et al.*
  📍 AAAI 2025
  > Large language models (LLMs) exhibit exceptional performance across various domains, and model editing holds significant potential to improve LLM safety and mitigate issues such as hallucinations.
...

**7. [MultiQ&A: An Analysis in Measuring Robustness via Automated Crowdsourcing of Question Perturbations and Answers](https://openreview.net/forum?id=3ytx6AoSBK)**
  *Nicole Cho, William Watson*
  📍 AAAI 2025
  > One critical challenge in the institutional adoption journey of Large Language Models (LLMs) stems from their propensity to hallucinate in generated responses. To address this, we propose MultiQ&A,...

**8. [WaterSeeker: Pioneering Efficient Detection of Watermarked Segments in Large Documents](https://openreview.net/forum?id=3dslkUEgJb)**
  *Leyi Pan, Aiwei Liu, Yijian LU et al.*
  📍 AAAI 2025
  > Watermarking algorithms for large language models (LLMs) have attained high accuracy in detecting LLM-generated text. However, existing methods primarily focus on distinguishing fully watermarked t...

**9. [Halu-J: Critique-Based Hallucination Judge](https://openreview.net/forum?id=PFNvlDbq2F)**
  *Binjie Wang, Steffi Chern, Ethan Chern et al.*
  📍 AAAI 2025
  > Large language models (LLMs) frequently generate non-factual content, known as hallucinations. Existing retrieval-augmented-based hallucination detection approaches typically address this by framin...

**10. [Learning to Generate Research Idea with Dynamic Control](https://openreview.net/forum?id=zCb0dPvGYN)**
  *Ruochen Li, Liqiang Jing, Xinya Du*
  📍 AAAI 2025
  > The rapid advancements in large language models (LLMs) have demonstrated their potential to accelerate scientific discovery, particularly in automating the process of research ideation. 
LLM-based ...

### ⭐ Spotlight

**11. [The Structure of Relation Decoding Linear Operators in Large Language Models](https://openreview.net/forum?id=XsBzmJzJ2l)**
  *Miranda Anna Christ, Adrián Csiszárik, Gergely Becsó et al.*
  📍 NeurIPS 2025 spotlight
  > This paper investigates the structure of linear operators introduced in Hernandez et al. [2023] that decode specific relational facts in transformer language models. We extend their single-relation...

**12. [HM3: Hierarchical Multi-Objective Model Merging for Pretrained Models](https://openreview.net/forum?id=JeP0lpusYw)**
  *Yu Zhou, Xingyu Wu, Jibin Wu et al.*
  📍 NeurIPS 2025 spotlight
  > Model merging is a technique that combines multiple large pretrained models into a single model, enhancing performance and broadening task adaptability without original data or additional training....

**13. [Twilight: Adaptive Attention Sparsity with Hierarchical Top-$p$  Pruning](https://openreview.net/forum?id=Ve693NkzcU)**
  *Chaofan Lin, Jiaming Tang, Shuo Yang et al.*
  📍 NeurIPS 2025 spotlight
  > Leveraging attention sparsity to accelerate long-context large language models (LLMs) has been of great importance recently. However, most existing sparse attention algorithms use a fixed budget of...

**14. [Towards Practical Defect-Focused Automated Code Review](https://openreview.net/forum?id=mEV0nvHcK3)**
  *Junyi Lu, Lili Jiang, Xiaojia Li et al.*
  📍 ICML 2025 spotlightposter
  > The complexity of code reviews has driven efforts to automate review comments, but prior approaches oversimplify this task by treating it as snippet-level code-to-text generation and relying on tex...

**15. [BaxBench: Can LLMs Generate Correct and Secure Backends?](https://openreview.net/forum?id=il3KRr4H9u)**
  *Mark Vero, Niels Mündler, Victor Chibotaru et al.*
  📍 ICML 2025 spotlightposter
  > Automatic program generation has long been a fundamental challenge in computer science. Recent benchmarks have shown that large language models (LLMs) can effectively generate code at the function ...

**16. [How new data permeates LLM knowledge and how to dilute it](https://openreview.net/forum?id=NGKQoaqLpo)**
  *Chen Sun, Renat Aksitov, Andrey Zhmoginov et al.*
  📍 ICLR 2025 Spotlight
  > Large language models continually learn through the accumulation of gradient-based updates, but how individual pieces of new information affect existing knowledge, leading to both beneficial genera...

**17. [CiteGuard: Retrieval-Augmented Citation Verification for LLM-Powered Peer Review](https://openreview.net/forum?id=TKmxf2oSSR)**
  *Ishaan Gangwani, Aayam Bansal*
  📍 NeurIPS2025-AI4Science Spotlight
  > Accurate citations are essential for reproducibility and cumulative scientific progress, yet citation errors remain common and rarely receive systematic scrutiny in automated reviewing workflows. W...

**18. [10 Million Particle Events: Enabling Foundation Models for Sparse 3D Inverse Problems](https://openreview.net/forum?id=N9xLKbMXKW)**
  *Omar Alterkait, Sam Young, Ka Vang Tsang et al.*
  📍 NeurIPS2025-AI4Science Spotlight
  > Next-generation particle physics experiments require unprecedented machine learning capabilities to achieve their science goals. We propose generating 10 million particle detector events, the first...

**19. [When Do LLMs Improve Bayesian Optimization? A Systematic Comparison Across Molecular and Protein Design](https://openreview.net/forum?id=x3Y2TwVXwE)**
  *Mattias Akke, Soojung Yang, Jurģis Ruža et al.*
  📍 NeurIPS2025-AI4Science Spotlight
  > While powerful, classical Bayesian Optimization (BO) and active learning methods struggle to incorporate complex prior knowledge, provide limited interpretability in explaining why a candidate look...

**20. [Activation Control for Efficiently Eliciting Long Chain-of-thought Ability of Language Models](https://openreview.net/forum?id=XNo4yS9n1k)**
  *Zekai Zhao, Qi Liu, Kun Zhou et al.*
  📍 NeurIPS 2025
  > Despite the remarkable reasoning performance, eliciting the long chain-of-thought(CoT) ability in large language models(LLMs) typically requires costly reinforcement learning or supervised fine-tun...

### 📌 Poster

**21. [Time-o1: Time-Series Forecasting Needs Transformed Label Alignment](https://openreview.net/forum?id=RxWILaXuhb)**
  *Hao Wang, Licheng Pan, Zhichao Chen et al.*
  📍 NeurIPS 2025 poster
  > Training time-series forecasting models poses unique challenges in loss function design. Most existing approaches adopt temporal mean squared error, but this study reveals two critical limitations:...

**22. [REVE: A Foundation Model for EEG - Adapting to Any Setup with Large-Scale Pretraining on 25,000 Subjects](https://openreview.net/forum?id=ZeFMtRBy4Z)**
  *Yassine El Ouahidi, Jonathan Lys, Philipp Thölke et al.*
  📍 NeurIPS 2025 poster
  > Foundation models have transformed AI by reducing reliance on task-specific data through large-scale pretraining. While successful in language and vision, their adoption in EEG has lagged due to th...

**23. [Vulnerable Data-Aware Adversarial Training](https://openreview.net/forum?id=yrrU5YChQr)**
  *Yuqi Feng, Jiahao Fan, Yanan Sun*
  📍 NeurIPS 2025 poster
  > Fast adversarial training (FAT) has been considered as one of the most effective alternatives to the computationally-intensive adversarial training. Generally, FAT methods pay equal attention to ea...

**24. [The Good, the Bad and the Ugly: Meta-Analysis of Watermarks, Transferable Attacks and Adversarial Defenses](https://openreview.net/forum?id=NVDrWBwJTV)**
  *Grzegorz Gluch, Berkant Turan, Sai Ganesh Nagarajan et al.*
  📍 NeurIPS 2025 poster
  > We formalize and analyze the trade-off between backdoor-based watermarks and adversarial defenses, framing it as an interactive protocol between a verifier and a prover. While previous works have p...

**25. [Improved Algorithms for Overlapping and Robust Clustering of Edge-Colored Hypergraphs: An LP-Based Combinatorial Approach](https://openreview.net/forum?id=F3DrgOZYc6)**
  *Changyeol Lee, Yongho Shin, Hyung-Chan An*
  📍 NeurIPS 2025 poster
  > Clustering is a fundamental task in both machine learning and data mining. Among various methods, edge-colored clustering (ECC) has emerged as a useful approach for handling categorical data. Given...

**26. [GoalLadder: Incremental Goal Discovery with Vision-Language Models](https://openreview.net/forum?id=BiowiwzQaO)**
  *Alexey Zakharov, Shimon Whiteson*
  📍 NeurIPS 2025 poster
  > Natural language can offer a concise and human-interpretable means of specifying reinforcement learning (RL) tasks. The ability to extract rewards from a language instruction can enable the develop...

**27. [Router-R1: Teaching LLMs Multi-Round Routing and Aggregation via Reinforcement Learning](https://openreview.net/forum?id=DWf4vroKWJ)**
  *Haozhen Zhang, Tao Feng, Jiaxuan You*
  📍 NeurIPS 2025 poster
  > The rapid emergence of diverse large language models (LLMs) has spurred the development of LLM routers that assign user queries to the most suitable model. However, existing LLM routers typically p...

**28. [Structure-Aware Spectral Sparsification via Uniform Edge Sampling](https://openreview.net/forum?id=Z4eFqgYbha)**
  *Kaiwen He, Petros Drineas, Rajiv Khanna*
  📍 NeurIPS 2025 poster
  > Spectral clustering is a fundamental method for graph partitioning, but its reliance on eigenvector computation limits scalability to massive graphs. Classical sparsification methods preserve spect...

**29. [From Euler to AI: Unifying Formulas for Mathematical Constants](https://openreview.net/forum?id=cNqMAmpZh4)**
  *Tomer Raz, Michael Shalyt, Elyasheev Leibtag et al.*
  📍 NeurIPS 2025 poster
  > The constant $\large \pi$ has fascinated scholars throughout the centuries, inspiring numerous formulas for its evaluation, such as infinite sums and continued fractions. Despite their individual s...

**30. [CVGL: Causal Learning and Geometric Topology](https://openreview.net/forum?id=1CqEAuRzHc)**
  *Songsong Ouyang, Yingying Zhu*
  📍 NeurIPS 2025 poster
  > Cross-view geo-localization (CVGL) aims to estimate the geographic location of a street image by matching it with a corresponding aerial image. This is critical for autonomous navigation and mappin...

### 🆕 Recent (ArXiv / OpenReview)

**31. [Explainable Machine Learning for Scientific Insights and Discoveries](https://openreview.net/forum?id=uq8Ksuz9cI)**
  *Ribana Roscher, Bastian Bohn, Marco F. Duarte et al.*
  📍 IEEE Access
  > Machine learning methods have been remarkably successful for a wide range of application areas in the extraction of essential information from data. An exciting and relatively recent development is...

**32. [Speech Analytics Based on Machine Learning](https://openreview.net/forum?id=LUDAVa3MYG)**
  *Grazina Korvel, Adam Kurowski, Bozena Kostek et al.*
  📍 Machine Learning Paradigms 2019
  > In this chapter, the process of speech data preparation for machine learning is discussed in detail. Examples of speech analytics methods applied to phonemes and allophones are shown. Further, an a...

**33. [Evaluation of Interactive Machine Learning Systems](https://openreview.net/forum?id=UZVbSNeFCU)**
  *Nadia Boukhelifa, Anastasia Bezerianos, Evelyne Lutton*
  📍 Human and Machine Learning 2018
  > The evaluation of interactive machine learning systems remains a difficult task. These systems learn from and adapt to the human, but at the same time, the human receives feedback and adapts to the...

**34. [Beyond Human-in-the-Loop: Empowering End-Users with Transparent Machine Learning](https://openreview.net/forum?id=SfbmBKpcNi)**
  *Patrick C. Shih*
  📍 Human and Machine Learning 2018
  > Advances in data analytics and human computation are transforming how researchers conduct science in domains like bioinformatics, computational social science, and digital humanities. However, data...

**35. [Opportunities for machine learning in scientific discovery](https://openreview.net/forum?id=XsCsNLiMsv)**
  *Ricardo Vinuesa, Jean Rabault, Hossein Azizpour et al.*
  📍 CoRR 2024
  > As modern scientific instruments generate vast amounts of data and the volume of information in the scientific literature continues to grow, machine learning (ML) has become an essential tool for o...

**36. [Machine Learning for Intelligent Information Access](https://openreview.net/forum?id=p23T2iWcuJ)**
  *Grigoris I. Karakoulas, Giovanni Semeraro*
  📍 Machine Learning and Its Applications 2001
  > As the volume of electronically stored information continues to expand across computer networks, the need for intelligent access to on-line collections of multimedia documents becomes imperative. E...

**37. [An Optimal Machine Learning Classification Model for Flash Memory Bit Error Prediction](https://openreview.net/forum?id=SAvNLoibAJ)**
  *Barry Fitzgerald, Conor Ryan, Joe Sullivan*
  📍 Machine Learning Paradigms 2019
  > NAND flash memory is now almost ubiquitous in the world of data storage. However, NAND wears out as it is used, and manufacturers specify the number of times a device can be rewritten (known as pro...

**38. [Active learning for data streams: a survey](https://openreview.net/forum?id=odwozdNaMi)**
  *Davide Cacciarelli, Murat Kulahci*
  📍 Machine Learning
  > Online active learning is a paradigm in machine learning that aims to select the most informative data points to label from a data stream. The problem of minimizing the cost associated with collect...

**39. [Interactive Machine Learning for Applications in Food Science](https://openreview.net/forum?id=fgTycjfgYu)**
  *Alberto Paolo Tonda, Nadia Boukhelifa, Thomas Chabin et al.*
  📍 Human and Machine Learning 2018
  > The apparent simplicity of food processes often hides complex systems, where physical, chemical and living organisms’ processes co-exist and interact to create the final product. Data can be plague...

**40. [SciMRC: Multi-perspective Scientific Machine Reading Comprehension](https://openreview.net/forum?id=sRQNyWMFsF)**
  *Xiao Zhang, Heqi Zheng, Yuxiang Nie et al.*
  📍 LREC/COLING 2024
  > Scientific Machine Reading Comprehension (SMRC) aims to facilitate the understanding of scientific texts through human-machine interactions. While existing dataset has significantly contributed to ...

**41. [Enhancing Scientific Discourse: Machine Translation for the Scientific Domain](https://openreview.net/forum?id=WB4GKzei1Y)**
  *Dimitris Roussis, Sokratis Sofianopoulos, Stelios Piperidis*
  📍 EAMT (1) 2024
  > The increasing volume of scientific research necessitates effective communication across language barriers. Machine translation (MT) offers a promising solution for accessing international publicat...

**42. [PDeepPP:A Deep learning framework with Pretrained Protein language for peptide classification](https://openreview.net/forum?id=pcBXOjxiIY)**
  *Jixiu Zhai, Tianchi Lu, Haitian Zhong et al.*
  📍 CoRR 2025
  > Advances in peptide identification are revolutionizing our ability to decipher protein functions and accelerate therapeutic discovery. We present PDeepPP, a deep learning framework that integrates ...

**43. [Torn between recommending acceptance and rejection; interesting architecture but multiple concerns](https://openreview.net/forum?id=dYDEpTKseq)**
  **
  > 

**44. [Initial review](https://openreview.net/forum?id=7hbH59PEbV)**
  **
  > 

**45. [PDE-CNNs: Axiomatic Derivations and Applications](https://openreview.net/forum?id=xHJ0VM0qrf)**
  *Gijs Bellaard, Sei Sakata, Bart M. N. Smets et al.*
  📍 CoRR 2024
  > PDE-based Group Convolutional Neural Networks (PDE-G-CNNs) use solvers of evolution PDEs as substitutes for the conventional components in G-CNNs. PDE-G-CNNs can offer several benefits simultaneous...

**46. [Nice benchmark for autoregressive models](https://openreview.net/forum?id=buPZqbJ8wU)**
  **
  > 

**47. [Free-Form Geometric Modeling by Integrating Parametric and Implicit PDEs](https://openreview.net/forum?id=yFziNFmpGM)**
  *Haixia Du, Hong Qin*
  📍 IEEE Trans. Vis. Comput. Graph. 2007
  > Parametric PDE techniques, which use partial differential equations (PDEs) defined over a 2D or 3D parametric domain to model graphical objects and processes, can unify geometric attributes and fun...

**48. [Thank you for your valuable review](https://openreview.net/forum?id=dT0d0qurhK)**
  **
  >
